static/openapi.json

```
@@ -11,17 +11,20 @@
                     "tasks"
                 ],
                 "summary": "Get Tasks",
-                "description": "Get a list of the IDs of all available labelling tasks",
+                "description": "Get a list of the IDs of all available labeling tasks",
                 "operationId": "get_tasks",
                 "responses": {
                     "200": {
                         "description": "Successful Response",
                         "content": {
                             "application/json": {
-                                "schema": {},
-                                "example": [
-                                    "ecg-qrs-classification-physiodb"
-                                ]
+                                "schema": {
+                                    "title": "Response Get Tasks",
+                                    "type": "array",
+                                    "items": {
+                                        "$ref": "#/components/schemas/Task"
+                                    }
+                                }
                             }
```

```suggestion
                            },
                            "example": [
                                  "ecg-qrs-classification-physiodb",
                                  "pressure-mat-pose-estimation-slp"
                            ]
```
No must have, just for completion

---

I dont know how to do that, because this file gets overriden when the api is regenerated. @bewee do you know where this example has to be added in fastapi? Additionally, isnt that an example for the case when just the ids were returned? In this pr, the whole task is returned now, but the concept remains of adding a list of whole tasks

---

Yes, definitely don't change this file by hand. Don't know how to add it to fastapi right now, let's have a look at it together tomorrow.